controller/users.go

```diff
@@ -254,6 +268,40 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 	})
 }
 
+func isUsernameUnique(appl application.Application, username string, identity account.Identity) (bool, error) {
+	usersWithSameUserName, err := appl.Identities().Query(account.IdentityFilterByUsername(username))
+	if err != nil {
+		log.Error(context.Background(), map[string]interface{}{
```

Comment:
Not sure how context.Background() works but why not to pass the context as a param of this function?